java/pom.xml

```diff
@@ -166,6 +166,27 @@
           <artifactId>maven-antrun-plugin</artifactId>
           <version>1.8</version>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
```

Comment:
This is great! Do you think there's any way we can enforce this with Bazel checks as well?

Reply:
I don't think there's an existing animalsniffer plugin for bazel. It should be possible to write one, though I doubt it's worth the effort.  Bazel can compile android libraries? Perhaps we could create a sample project whose compile would fail if the api level went too high.

Reply:
Yes there is android_(library|binary) and it is actively a use case people use, if we're verifying this in Maven I'd like to see parity in Bazel given we have 1:1 parity with all other tests.